pkg/pool-stable/test/foundry/FungibilityStable.t.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity ^0.8.24;
+
+import "forge-std/Test.sol";
+
+import { IVault } from "@balancer-labs/v3-interfaces/contracts/vault/IVault.sol";
+import "@balancer-labs/v3-interfaces/contracts/vault/VaultTypes.sol";
+
+import { CastingHelpers } from "@balancer-labs/v3-solidity-utils/contracts/helpers/CastingHelpers.sol";
+
+import { PoolHooksMock } from "@balancer-labs/v3-vault/contracts/test/PoolHooksMock.sol";
+import { ProtocolFeeControllerMock } from "@balancer-labs/v3-vault/contracts/test/ProtocolFeeControllerMock.sol";
+import { FungibilityTest } from "@balancer-labs/v3-vault/test/foundry/Fungibility.t.sol";
+
+import { StablePoolFactory } from "../../contracts/StablePoolFactory.sol";
+import { StablePool } from "../../contracts/StablePool.sol";
+import { StablePoolContractsDeployer } from "./utils/StablePoolContractsDeployer.sol";
+
+contract FungibilityStableTest is StablePoolContractsDeployer, FungibilityTest {
+    using CastingHelpers for address[];
+
+    uint256 internal constant DEFAULT_AMP_FACTOR = 200;
+
+    /// @notice Overrides BaseVaultTest _createPool(). This pool is used by FungibilityTest.
+    function _createPool(address[] memory tokens, string memory label) internal override returns (address) {
+        StablePoolFactory factory = deployStablePoolFactory(IVault(address(vault)), 365 days, "Factory v1", "Pool v1");
+        PoolRoleAccounts memory roleAccounts;
+
+        // Allow pools created by `factory` to use poolHooksMock hooks.
+        PoolHooksMock(poolHooksContract).allowFactory(address(factory));
+
+        StablePool newPool = StablePool(
+            factory.create(
+                "Stable Pool",
+                "STABLE",
+                vault.buildTokenConfig(tokens.asIERC20()),
+                DEFAULT_AMP_FACTOR,
+                roleAccounts,
+                swapFeePercentage, // 1% swap fee, but test will override it
```

Comment:
I think in this case it doesn't :) (see comment above)

It's probably not a big deal because the taxable amount will be minimal anyways, but the test should just use 0 fees.